content/pt/docs/reference/tools.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ferramentas
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+O Kubernetes contém várias ferramentas internas para ajudá-lo a trabalhar com o sistema Kubernetes.
```

Comment:
Maybe is better change to: "O Kubernetes **possui** várias ferramentas internas para ajudá-lo a trabalhar com o sistema Kubernetes."